dada_tests/interpret/ops/op_neg_uint.dada

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    -1u
+#! ^^^^ RUN ERROR cannot apply operator - to an unsigned integer
+# FIXME: Error highlighting should not include the leading whitespace
```

Comment:
The span of the unary expression includes the leading whitespace, which is weird

Reply:
I think I have fixed it